content/blog/multi-lang-components/index.md

```diff
@@ -14,29 +14,31 @@ tags:
     - typescript
 ---
 
-Today we're excited to announce support for one of our most highly-requested features: multi-language components! Our
-first multi-language component is the popular [EKS package], which was previously only available for TypeScript. With the
-[v0.22.0 release], teams can [manage EKS clusters] in their language of choice (Python, .NET, Go, and TypeScript).
+Pulumi's infrastructure as code tooling combines the programming languages and tools you already know with the full power of cloud
+infrastructure. But, until now, some of that Pulumi components for that cloud infrastructure, like our popular [EKS package], were
+only available in a subset of the languages supported by Pulumi.
 
-<!--more-->
+Now, with multi-language components, anyone can author a Pulumi component once, in the language they prefer, and make it available
+across all Pulumi languages. To celebrate, we've taken that [EKS package]–previously only available for TypeScript–
+and brought it to all four Pulumi languages: TypeScript, Python, .NET, and Go. Now, regardless of the language you choose, you can
+[manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the EKS package]. Check out our Modern Infrastructure
+Wednesday video to see it in action:
 
-Pulumi's support for many familiar programming languages means that your organization is free to choose which supported
-language works best for each team. It's already possible to collaborate across languages by using separate Pulumi projects
-and referring to them using [Stack References]. Multi-language components take this one step further towards the vision of
-"write once, use anywhere". Component authors broaden their reach across languages, and users benefit from the work
-of a much larger community.
+{{< youtube "gxLyAr0lUg0?rel=0" >}}
 
-Multi-language component support allows component authors to project their API into all Pulumi-supported languages
-idiomatically and without requiring knowledge of all of these languages. First-class authoring support for multi-language
-components is in the works.
+<!--more-->
```

Comment:
is this cut line necessary?

Reply:
I figured we had to have one for the blog to render well - will defer to you on what's best here.

Reply:
I'll merge it and see how it renders in Hugo